generator/src/main/java/org/stjs/generator/javac/ElementUtils.java

```diff
@@ -445,4 +445,23 @@ public static boolean hasMultipleConstructors(Element element) {
 		return constructorCount > 1;
 	}
 
+	public static boolean isClass(Element element) {
+		return (element != null) && (element.getKind() == ElementKind.CLASS);
+	}
+
+	public static boolean isInnerClass(Element element) {
+		return isClass(element) && isClass(element.getEnclosingElement());
```

Comment:
In the previous `isInnerClass()` method, they were making this check:
```java
	if (ElementKind.CLASS.equals(element.getKind())) {
		String typeName = getTypeNameForElement(context, element);
		return typeName.contains(".");
	}
```

Is there a reason why it was dropped?